packages/web-api/src/WebClient.ts

```diff
@@ -645,3 +647,18 @@ function warnIfFallbackIsMissing(method: string, logger: Logger, options?: WebAP
     }
   }
 }
+
+/**
+ * Log a warning when thread_ts is not a string
+ * @param method api method being called
+ * @param logger instance of web clients logger
+ * @param options arguments for the Web API method
+ */
+function warnIfThreadTsIsNotString(method: string, logger: Logger, options?: WebAPICallOptions): void {
+  const targetMethods = ['chat.postEphemeral', 'chat.postMessage', 'chat.scheduleMessage', 'files.upload'];
+  const isTargetMethod = targetMethods.includes(method);
+
+  if (isTargetMethod && options?.thread_ts !== undefined && typeof options?.thread_ts !== 'string') {
+    logger.warn('The thread_ts argument need to be a string.');
```

Comment:
Can you include the method name in the log message like we do in `warnIfFallbackIsMissing` method?

```suggestion
    logger.warn(`The given \`thread_ts\` value in the request payload for a ${method} call is a float value. We highly recommend using a string value instead.');
```

Reply:
@seratch Thank you for your review. I fixed the thing you suggested.